docs/development/testguide.rst

```diff
@@ -680,15 +680,51 @@ block::
 Image tests with pytest-mpl
 ===========================
 
+Running image tests
+-------------------
+
 We make use of the `pytest-mpl <https://pypi.python.org/pypi/pytest-mpl>`_
-plugin to create tests where we can compare the output of plotting commands
-with reference files (this is used for instance in
+plugin to write tests where we can compare the output of plotting commands
+with reference files on a pixel-by-pixel basis (this is used for instance in
 :ref:`astropy.visualization.wcsaxes <wcsaxes>`).
 
 To run the Astropy tests with the image comparison, use::
 
     python setup.py test -a "--mpl" --remote-data
 
+However, note that the output can be very sensitive to the version of Matplotlib
+as well as all its dependencies (e.g. freetype), so we recommend running the
+image tests inside a `Docker <https://www.docker.com/>`__ container which has a
```

Comment:
maybe add a link to the astropy repository for the containers, too (not necessary here, but somewhere in this paragraph)

Reply:
Done